packages/test-server/package.json

```
@@ -0,0 +1,68 @@
+{
+    "name": "@ai-sdk/test-server",
+    "version": "1.0.0-beta.0",
```

```suggestion
    "version": "0.0.0",
```

I verified now that this is what we want. Then in the `.changeset` file use `major` for the initial release of a package

---

understood, changed!